docs/index.md

```diff
@@ -3,7 +3,7 @@ Content Creator Integration - Documentation
 
 Welcome to the official documentation for [Content Creator Integration](https://www.curseforge.com/minecraft/mc-mods/content-creator-integration/).
 
-This documentation was written based off of version 1.9.0. These docs are aimed to be a reference or a guide for people trying to understand how Content Creator Integration works.
+This documentation was written based off of version 1.10.0. These docs are aimed to be a reference or a guide for people trying to understand how Content Creator Integration works.
```

Comment:
I'm tempted to say something along the lines of "This documentation was written based off of version 1.9.0, with some updates up to version 1.10.0" to imply that docs are basically still for 1.9.0 despite being updated for 1.10.0, if you know what I mean?